Extension/src/LanguageServer/ui.ts

```diff
@@ -344,6 +344,34 @@ export class UI {
         return (selection) ? selection.index : -1;
     }
 
+    public async showSelectCompiler(paths: string[]): Promise<number> {
+        const options: vscode.QuickPickOptions = {};
+        options.placeHolder = localize("select.compile.commands", "Select a compiler to configure for IntelliSense");
```

Comment:
Maybe something like `Select a compiler to configure for IntelliSense (that you trust to execute)`? Otherwise, it's not obvious that when it's chosen that our extension will assume it's trustable to execute.

Reply:
This isn't meant to be exposed as a security feature but rather a configuration feature. From what I understand.

Reply:
Sure, I'm fine with this wording, but see my other comment (about only showing trusted compilers).